.github/workflows/pre-commit.yml

```
@@ -3,12 +3,11 @@ name: pre-commit
 on:
   pull_request:
     branches:
-      - "15.0*"
       - "15st*"
   push:
     branches:
-      - "15.0"
-      - "15st"
+      - "15st1"
+      - "15.0-ocabot-*"
```

```suggestion
      - "15.0*"
      - "15st1"
      - "15.0-ocabot-*"
```
In view of switching the master branch from 15st1 to 15.0.  Once the switch is done we will remove the line of 15st1.